.build-script

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#====================#
+# Altis Build Script #
+#====================#
+
+# Install composer dependencies
+composer install --no-dev
+
+# Edit below this line to add any additional build steps.
+#
+# Available binaries:
```

Comment:
Can we link to the docs for something more evergreen?

Reply:
Oh yeah good idea!

Reply:
so useful having the site live :)